qiskit/opflow/primitive_ops/pauli_op.py

```diff
@@ -99,11 +99,14 @@ def _expand_dim(self, num_qubits: int) -> 'PauliOp':
     def tensor(self, other: OperatorBase) -> OperatorBase:
         # Both Paulis
         if isinstance(other, PauliOp):
-            # Copying here because Terra's Pauli kron is in-place.
-            op_copy = Pauli((other.primitive.z, other.primitive.x))
```

Comment:
I'm a bit confused by this comment, but it's not needed as `tensor` is not inplace, right?

Reply:
Sorry. I forgot the description. 
Yes. kron is deprecated now and tensor is not inplace. https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/quantum_info/operators/symplectic/pauli.py#L779-L782
expand and tensor differs the direction https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/quantum_info/operators/symplectic/base_pauli.py#L65-L69